### DISCUSSION TITLE
Ctrl+m and Ctrl+j bindings don't work

### DISCUSSION THREAD

**Discussion by txtyash** [zellij-org/zellij#1512] [General]
I wished to bind "Ctrl+M" to "Move" mode but unable to bind "Ctrl+M", as well as "Ctrl+J". I was able to bind "Ctrl+K" successfully.

I did check if the "Ctrl+M" and "Ctrl+J" were already bound in the terminal or fish shell, but they weren't. Is it because these bindings are interpreted as "Enter" by the terminals?

**Comment by cassepipe** [comment]
I am  also interested to know if there's a way to do that

**Comment by semi710** [comment]
> I wished to bind "Ctrl+M" to "Move" mode but unable to bind "Ctrl+M", as well as "Ctrl+J". I was able to bind "Ctrl+K" successfully.
> 
> I did check if the "Ctrl+M" and "Ctrl+J" were already bound in the terminal or fish shell, but they weren't. Is it because these bindings are interpreted as "Enter" by the terminals?

Yes the reason is same as these bindings returns carriage return (enter in the terminal)
It is an issue in zellij and have been labeled as bug.

**Reply by cassepipe** [reply]
Thanks for your reply
